lms/djangoapps/courseware/views/views.py

```diff
@@ -267,12 +268,16 @@ def courses(request):
     # Add marketable programs to the context.
     programs_list = get_programs_with_type(request.site, include_hidden=False)
 
+    # Get upcoming webinars to display them on `learn` page
+    upcoming_webinars = Webinar.objects.filter(status=Webinar.UPCOMING).order_by('start_time')
+
```

Comment:
No need for this variable can be moved directly at line 280 to avoid core changes at 2 places.

Reply:
Sure, I'll do that while creating PR for master.

Reply:
Create a PR for develop as well to merge that commit in develop as well.